### PR TITLE
feat: smb2 dashboard

### LINKED ISSUE #1696: Implement SMB2 panels in Harvest
Thanks @Watson9121 for raising this via [discord](https://discord.com/channels/855068651522490400/1072491206912778260).

Investigate and implement SMB2 view in Harvest.

### PR DESCRIPTION
**Pull request by rahulguptajss**
Fixes #1696

image

image

image

### CODE REVIEW

**Comment by cgrinds** [comment]
These changes look good, few general comments:

- Not immediately clear these should be enabled by default. Let's discuss in our meeting
- The panels probably need an SVM or cluster prefix for when multiples are selected in top vars
- We need to ask for these to be added to rest perf too

File: grafana/dashboards/cmode/smb2.json [hunk 1]
```diff
@@ -0,0 +1,1290 @@
...
+          "exemplar": false,
+          "expr": "sum(smb2_read_ops{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\"})",
+          "interval": "",
+          "legendFormat": "read_ops",
+          "refId": "A"
+        },
+        {
+          "exemplar": false,
+          "expr": "sum(smb2_read_ops{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\"})",
+          "hide": false,
```
Comment (Hardikl) [inline grafana/dashboards/cmode/smb2.json:177]: write_ops

File: grafana/dashboards/cmode/smb2.json [hunk 2]
```diff
@@ -0,0 +1,1290 @@
...
+          "format": "heatmap",
+          "instant": false,
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "{{ le }}",
+          "refId": "A"
+        }
+      ],
+      "timeFrom": null,
+      "title": "Lock Latency HistogramHeatmap",
```
Comment (Hardikl) [inline grafana/dashboards/cmode/smb2.json:857]: Histogram Heatmap

File: grafana/dashboards/cmode/smb2.json [hunk 3]
```diff
@@ -0,0 +1,1290 @@
...
+          "format": "heatmap",
+          "instant": false,
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "{{ le }}",
+          "refId": "A"
+        }
+      ],
+      "timeFrom": null,
+      "title": "Session Setup Latency Heatmap",
```
Comment (Hardikl) [inline grafana/dashboards/cmode/smb2.json:1133]: name and query doesn't match.

**Review by Hardikl — APPROVED** [review]

File: grafana/dashboards/cmode/smb2.json [hunk 4]
```diff
@@ -375,7 +375,7 @@
       "targets": [
         {
           "exemplar": false,
-          "expr": "smb2_read_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$TopSMB2ReadLatency\"}",
+          "expr": "smb2_read_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",node=~\"$Node\",svm=~\"$TopSMB2ReadLatency\"}",
```
Comment (Hardikl) [inline grafana/dashboards/cmode/smb2.json:378]: topk required?

File: grafana/dashboards/cmode/smb2.json [hunk 5]
```diff
@@ -461,7 +461,7 @@
       "targets": [
         {
           "exemplar": false,
-          "expr": "smb2_write_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$TopSMB2WriteLatency\"}",
+          "expr": "smb2_write_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",node=~\"$Node\",svm=~\"$TopSMB2WriteLatency\"}",
```
Comment (Hardikl) [inline grafana/dashboards/cmode/smb2.json:464]: topk?

**Review by Hardikl — APPROVED** [review]

**Review by cgrinds — DISMISSED** [review]

**Review by cgrinds — APPROVED** [review]

**Review by Hardikl — APPROVED** [review]

**Pull request merged**